GiEnJul/Models/Recipient.cs

```diff
@@ -5,6 +5,13 @@ namespace GiEnJul.Models
 {
     public class Recipient : TableEntity
     {
+
+        public Recipient(string location, string rowKey) : base(location, rowKey)
+        {
+            PartitionKey = $"loc_{location}";
+            RowKey = rowKey;
+        }
```

Comment:
This constructor seems to be unused
```suggestion
```

Reply:
That is true. Used it for testing and forgot to remove it